packages/be-gateway/src/routes/task/index.ts

```diff
@@ -367,6 +375,10 @@ router.put('/project/task-many', async (req: AuthRequest, res) => {
     await mdTaskUpdateMany(ids, data)
     await findNDelCaches(key)
     logging.info('TaskUpdateMany - successfully')
+    taskPusherJob.triggerUpdateEvent({
```

Comment:
đoạn này viết đúng nhưng nhầm chỗ rồi e ạ
toàn bộ controller chỉ kiểm soát việc nhận body, param và trả về client thôi
còn business logic thì nằm hết trong service
nên đoạn code trên chuyển toàn bộ vào bên trong services/task/update.service.ts